src/main/java/website/frontrow/util/FileNameCollector.java

```diff
@@ -50,4 +61,42 @@ public void addToLog(String action)
     {
         Log.add(action);
     }
+
+    private ArrayList<String> loadFilesFromJar(JarFile jar, String folder)
```

Comment:
It would be nice if we could generalize this. (Maybe using an iterator.)
But still, the IDE loading is pretty much dev-env only, maybe there is another solution.

No haste with this one, as it seems to be a complex task, which may or may not be easy to do.